src/components/button-minor/button-minor.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ButtonMinor from ".";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+
+test("should render with children", () => {
+  render(<ButtonMinor>foo</ButtonMinor>);
+
+  expect(screen.getByRole("button", { name: "foo" })).toBeVisible();
+});
+
+test("should render disabled button when `disabled` is true", () => {
+  render(<ButtonMinor disabled>foo</ButtonMinor>);
+
+  expect(screen.getByRole("button", { name: "foo" })).toBeDisabled();
+});
+
+// coverage
+test("should render with expected styles when the button is icon only", () => {
+  render(<ButtonMinor iconType="bin" />);
+
+  expect(screen.getByTestId("icon")).toHaveStyle({ position: "absolute" });
+});
+
+// coverage
+test("should render with expected styles when `buttonType` is 'primary'", () => {
+  render(<ButtonMinor buttonType="primary">foo</ButtonMinor>);
+
+  const button = screen.getByRole("button", { name: "foo" });
+
+  expect(button).toHaveStyleRule("background", "var(--colorsActionMinor500)");
+  expect(button).toHaveStyleRule(
+    "border-color",
+    "var(--colorsActionMinorTransparent)"
+  );
+  expect(button).toHaveStyleRule("color", "var(--colorsActionMinorYang100)");
+});
+
+// coverage
+test("should render with expected styles when `buttonType` is 'tertiary'", () => {
```

Comment:
**comment (non-blocking)**: I'm puzzled that we're testing `primary` and `tertiary`, but not `secondary`. Especially when `secondary` is the default. Could we not add another identical test but against the styles for a secondary button, without the `buttonType` prop being passed. (And optionally do the same but with the prop explicitly set to secondary - that way we know we're testing all values even if the default ever changes.)

Reply:
As these are mainly for coverage and styles are being captured in chromatic, I didn't think that adding a test `secondary` variant would add much value, same with the `medium` size. Happy to add the tests if the second reviewer also thinks they should be included.